src/main/java/uk/gov/di/ipv/cri/experian/App.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.di.ipv.cri.experian;
+
+public class App {
+    public static void main(String[] args) {
+        System.setProperty(
+                "javax.net.ssl.keyStore", System.getenv("EXPERIAN_CROSS_CORE_API_KEYSTORE_PATH"));
```

Comment:
I don't think there's a way we can set the keystore on the filesystem for PaaS. For  the current KBV PaaS deploy, we base64 encode the keystore and set that as an env var, see https://github.com/alphagov/di-ipv-cri-kbv-experian-back/blob/paas/kbv-api/src/main/java/uk/gov/di/ipv/cri/kbv/KBVApp.java. We could do something similar here?

Reply:
I've amended to load the keystore from a (base64) environment variable (as suggested)